css/style.css

```diff
@@ -184,40 +204,81 @@ p {
 }
 
 /* Sales Page Styles */
+.formlist li {
+  display: inline-block;
+  margin-top: 5px;
+  margin-bottom: 5px;
+  padding-left: 20px;
+  min-width: 235px;
+  justify-content: space-around;
+}
+
 form {
   margin: 10px 10px 0;
+  font-family: "Open Sans", sans-serif;
+  border: 1px solid #202b19;
+  border-radius: 8px;
 }
 
 fieldset {
   border: none;
-  margin: 20px 0; /* don't need the unit on the 0 */
+  margin: 5px 0;
+}
+
+legend {
+  font-size: 20px;
+  font-weight: bold;
+  text-align: center;
+  padding-left: 20px;
 }
 
-label[for="minCustsPerHour"] {
-  display: block;
+label {
+  font-size: 12px;
+  text-transform: uppercase;
+  width: 80px;
 }
 
 /* attribute selector...INCREDIBLY useful!*/
 input[type="text"], input[type="number"], input[type="tel"] {
```

Comment:
@elaewin AGREED!  the attribute selector comes in even more handy when you get to 301 - we'll be using it a lot to query the DOM through the use of jQuery